src/lib/components/DatePickerPanel.js

```diff
@@ -102,6 +115,14 @@ export default class DatePickerPanel extends React.Component {
     });
   }
 
+  validateOnTimeChange() {
+    const { currentValue, confirmDisabled } = this.state;
+    const newConfirmDisabled = (currentValue.valueOf() <= Date.now());
```

Comment:
Maybe we should check that the current value is less than five seconds from now, to give ourselves a bit of a buffer?